parsing/dialects/dialect1/parserImpls.ftl

```diff
@@ -2382,3 +2382,164 @@ SqlCreateJoinIndex SqlCreateJoinIndex() :
             select, indices);
     }
 }
+
+SqlCall CaseN() :
```

Comment:
nit: Can we use the more specific SqlCaseN return type?

Reply:
Fixed in https://github.com/googleinterns/calcite/pull/212/commits/2ac5c3010b51f82c19624c1a7aadff8015134548